_posts/2015-03-23-design-patterns-part-1.markdown

```diff
@@ -0,0 +1,131 @@
+---
+layout:post
```

Comment:
So apparently spacing matters here. Can you put a space between 'layout:' and 'post'? Same goes for the other ones below.